plugins/modules/clustering/znode.py

```diff
@@ -122,7 +141,9 @@ def main():
             op=dict(choices=['get', 'wait', 'list']),
             state=dict(choices=['present', 'absent']),
             timeout=dict(default=300, type='int'),
-            recursive=dict(default=False, type='bool')
+            recursive=dict(default=False, type='bool'),
+            scheme=dict(default='digest', choices=['digest', 'sasl']),
+            credential=dict(type='str')
```

Comment:
A trailing comma will reduce the diff when adding another option. Also `no_log=True` will prevent credentials to be logged openly:
```suggestion
            credential=dict(type='str', no_log=True),
```